README.rst

```diff
@@ -36,6 +36,12 @@ In order to authorize against Avaya OneCloud™️ CPaaS  services you'll have t
     
 .. code-block:: bash
 
+You can also change the base API url in /constants.go. The default is set to "https://api.zang.io".
```

Comment:
Looks `/v2` is missing.

Reply:
Adhering to the value as mentioned here: https://github.com/zang-cloud/zang-go/blob/master/constants.go

Reply:
ok, but for new US and EU, are we using `/v2`

Reply:
yes we do.